### PR TITLE
Update cpu & memory resource recommendation

### PR DESCRIPTION
**Pull request by lefttree**
For context of making this change, please see [Jobs runner instability research](https://docs.google.com/document/d/1N8Ahnpz5sLUQFX7yJYoyrjN7x37rwqTy6nnbR-FkBXU/edit)

### CODE REVIEW

File: values.yaml [hunk 1]
```diff
@@ -198,6 +198,10 @@ extraVolumeMounts: []
 extraVolumes: []
 
 resources: {}
+  # If you have more than 1 replica, the minimum recommended resources configurate are as follows:
```
Comment (JatinNanda) [inline values.yaml:201]: configurate are as follows: -> configuration is as follows:?

File: values.yaml [hunk 2]
```diff
@@ -198,6 +198,10 @@ extraVolumeMounts: []
...
+  # - cpu: 2048m
+  # - memory: 4098Mi
```
Comment (JatinNanda) [inline values.yaml:203]: it's unclear whether these are requests or limits - also if this is what we suggest, can we just modify the existing 500m 500mi comments below?

Comment (lefttree) [reply values.yaml]: These should be requests. I updated the following requests and doubled the limit on top of that

**Review by JatinNanda — APPROVED** [review]
seems fine, but I'm worried this is a huge jump - could you tldr me how you arrived at these numbers?

**Pull request merged**